backend/btrixcloud/emailsender.py

```diff
@@ -58,10 +58,10 @@ def _send_encrypted(self, receiver: str, name: str, **kwargs) -> None:
         else:
             raise HTTPException(status_code=500, detail="invalid_email_template")
 
-        print(full, flush=True)
+        # print(full, flush=True)
 
         if not self.smtp_server:
-            print("Email: No SMTP Server, not sending", flush=True)
+            # print("Email: No SMTP Server, not sending", flush=True)
```

Comment:
Seems worth keeping uncommented

Reply:
Yes, this was mostly for test logs, but probably should be an option?

Reply:
Yeah that makes sense. Similarly, printing out the full email is very useful for dev but just clutter in tests.

Reply:
Added an option to turn the full email logs on/off:
https://github.com/webrecorder/browsertrix/pull/1914/commits/fb8337c881797449a79fd46e64a0256bcac44bcc